configs/app/features/getGasButton.ts

```diff
@@ -0,0 +1,36 @@
+import type { Feature } from './types';
+import type { GasRefuelProviderConfig } from 'types/client/gasRefuelProviderConfig';
+
+import { getEnvValue, parseEnvJson } from '../utils';
+import marketplace from './marketplace';
+
+const value = parseEnvJson<GasRefuelProviderConfig>(getEnvValue('NEXT_PUBLIC_GAS_REFUEL_PROVIDER_CONFIG'));
+
+const title = 'Get gas button';
+
+const config: Feature<{
+  name: string;
+  logoUrl?: string;
+  urlTemplate: string;
+  dappId?: string;
+  usdThreshold: number;
+}> = (() => {
+  if (value) {
+    return Object.freeze({
+      title,
+      isEnabled: true,
+      name: value.name,
+      logoUrl: value.logo,
+      urlTemplate: value.url_template,
+      dappId: marketplace.isEnabled ? value.dapp_id : undefined,
+      usdThreshold: value.usd_threshold || 1,
```

Comment:
It's not required in type definition 🙂

Reply:
<img width="213" alt="image" src="https://github.com/user-attachments/assets/c1597a76-49a6-460d-bbf9-8bed6fca0479">
it is

Reply:
This is the type of return value 🧐

Okay, I'll change it if you insist. The task stated that for all networks this threshold would be $1, and only for Ethereum it would be $10. That's why I decided to make $1 the default value just in case and for simplicity.